src/Mod/AddonManager/AddonManagerOptions.ui

```diff
@@ -232,6 +232,42 @@ of the line after a space (e.g. https://github.com/FreeCAD/FreeCAD master).</str
      </layout>
     </widget>
    </item>
+   <item>
+    <layout class="QHBoxLayout" name="horizontalLayout_2">
+     <item>
+      <widget class="QLabel" name="fclabel">
+       <property name="text">
+        <string>Python executable:</string>
```

Comment:
Since this is optional, I would indicate it to the user somehow, like Python executable(optional): or in the tooltip...

Reply:
Good idea, I did both.